ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/serde/json/ValueSpecJsonSchemaSerdeSupplierTest.java

```diff
@@ -33,6 +37,8 @@ public void shouldSerializeAndDeserializeDecimalsWithOutStrippingTrailingZeros()
 
     when(srClient.getLatestSchemaMetadata("t-value"))
         .thenReturn(new SchemaMetadata(0, 1, ""));
+    when(srClient.registerWithResponse(anyString(), any(), anyBoolean()))
```

Comment:
Ideally we'd fix this by switching away from `mock(SchemaRegistryClient)` to `MockSchemaRegistryClient`, the same as in https://github.com/confluentinc/ksql/pull/9982, but since this is for an older branch I don't feel as strongly. The costs to not doing that are (1) inconsistency within the codebase and (2) if another new method is added to SchemaRegistryClient later on, then we'll have to make another analogous change again in the future.